src/packet.c

```
@@ -3,295 +3,307 @@
         }
+    }
 
-        /* Return 0 to indicate "for us, don't forward" */
+    packet_state.replay_trackers[evict].source_id = source_id;
+    packet_state.replay_trackers[evict].highest_seen_seq = 0;
+    packet_state.replay_trackers[evict].window_bits = 0ULL;
+    *out_idx = evict;
+    return 0;
+}
+
+int packet_is_replay(uint16_t source, uint32_t seq) {
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'packet_is_replay' of convertible types are easily swapped by mistake

(bugprone-easily-swappable-parameters)